packages/shared/src/components/auth/ActiveUsersCounter.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactElement } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import { UserIcon } from '../icons';
+import { IconSize } from '../Icon';
+import { getRandomUsersCount } from '../../lib/activity';
+import { generateQueryKey, RequestKey } from '../../lib/query';
+
+export function ActiveUsersCounter(): ReactElement {
+  const { data } = useQuery(
```

Comment:
Intentionally applied `useQuery` to have a more realistic feel when the user goes out of the screen and then comes back by refreshing the count on window focus.

Manually just using the function would result in flicker since our component re-renders multiple times on mount.